### ISSUE TITLE
Rooms aren't being marked as read anymore

### ISSUE THREAD

**Issue by turt2live** [vector-im/riot-web#9137]
I'm stuck staring at a read marker that won't go away and the room remains bold after viewing it. This is unquestionably annoying.

**Comment by turt2live** [comment]
TLDR: Cannot reproduce under reasonable circumstances.

After investigating this further I believe the bug that was causing this has been fixed, although I can't narrow down where or when. I can still reproduce it under some circumstances, although the steps are unclear - the ease of reproducing the bug is no longer possible though. 

The read receipt (and read marker) are harder to trigger than before, however I don't think that's harmful enough to warrant a fix. Previously simply brushing your mouse over the app would cause it to mark messages as read, even if the app wasn't active. This was annoying (but also useful), but definitely something that has been fixed and probably for the better.

As a few data points:
* My test account is completely unable to reproduce this case
* My main account is able to reproduce the case, however under rare and unknown circumstances. Possible server performance is at play here, as the issue resolves itself after a small amount of frustration.
* My main account was previously seeing it on nearly every room, however in the last day or two the issue has almost entirely gone away.

Will reopen or open a new issue if it comes back, taking a look at server performance instead. Simulated server performance degradation did not appear to cause the problem reliably.
